[DOC-820] Clarify command usage for "promote_auto_flags", "upgrade_ysql" and "finalize_upgrade" (#27759)
yugabyte/yugabyte-db: docs/content/v2025.1/admin/yb-admin.md

    upgrade_ysql
```

> Running this command is an online operation and doesn't require stopping a running cluster. Note that `upgrade_ysql` is a cluster-level operation; you don't need to run it on every node.

Reviewer: ```suggestion
Running `upgrade_ysql` is an online operation and doesn't require stopping a running cluster. `upgrade_ysql` is also a cluster-level operation; you don't need to run it on every node.
```
